<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleForm</class>
 <widget class="QWidget" name="ConsoleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="txtConsole">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacerConsole">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,0">
     <item>
      <widget class="ComboBox" name="cboCommand">
       <property name="styleSheet">
        <string notr="true">padding-top: 3;
padding-bottom: 3;
padding-right: 8;</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>5</number>
       </property>
       <property name="maxCount">
        <number>10</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCommandSend">
       <property name="toolTip">
        <string>Send command</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/images.qrc">
         <normaloff>:/images/send_1.png</normaloff>:/images/send_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdClearConsole">
       <property name="toolTip">
        <string>Clear console</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/images.qrc">
         <normaloff>:/images/erase_1.png</normaloff>:/images/erase_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">View/Widgets/ComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
